### PR TITLE
[WIP] Enable doc generation

### LINKED ISSUE #3095: Reactivate docs by using FSF as tool
Currently we don't publish docs - see https://github.com/fsprojects/FSharp.Formatting/issues/461

@matthid sugested to use FSharp.Formattings as command line tool

### PR DESCRIPTION
**Pull request by wallymathieu**
- [ ] Should work on CI
- [ ] Should be able to open Paket.Core in Visual Studio
- Upgraded to latest fsharp formatting 

This fixes #3095 (it looks like FAKE is used as the tool)

### CODE REVIEW

**Comment by forki** [comment]
hey. ok this allows me to create docs, but in VS I can no longer load the Paket.Core project

**Comment by wallymathieu** [comment]
What happens if you replicate the steps I took? 

- change build.fsx to have enabled docs and make fake path absolute
- run paket install 

Do you get the same behavior?

File: src/Paket.Core/Paket.Core.fsproj [hunk 1]
```diff
@@ -60,17 +60,17 @@
...
     <Compile Include="Common\AssemblyInfo.fs" />
     <Compile Include="Common\CustomAssemblyInfo.fs" />
-    <Compile Include="..\..\paket-files\fsprojects\FSharp.TypeProviders.StarterPack\src\AssemblyReader.fs">
-      <Paket>True</Paket>
-      <Link>Common/AssemblyReader.fs</Link>
-    </Compile>
     <Compile Include="..\..\paket-files\fsharp\FAKE\src\app\FakeLib\Globbing\Globbing.fs">
       <Paket>True</Paket>
-      <Link>Common/Globbing.fs</Link>
+      <Link>Common//Globbing.fs</Link>
```
Comment (wallymathieu) [inline src/Paket.Core/Paket.Core.fsproj]: why has it been added as `Common//Globbing.fs` ?

Comment (forki) [reply src/Paket.Core/Paket.Core.fsproj]: can you revert that? doesn't look good

Comment (wallymathieu) [reply src/Paket.Core/Paket.Core.fsproj]: I can revert it. Weird that it was added.

Comment (wallymathieu) [reply src/Paket.Core/Paket.Core.fsproj]: I'm using Mac OS, vs code, et.c. to edit the files. Could that be why?

**Comment by forki** [comment]
Thanks!

**Pull request merged**